torchmultimodal/modules/layers/attention.py

```diff
@@ -181,10 +181,10 @@ def __init__(
         self.d_qk = dim_q // n_head
         self.d_v = dim_kv // n_head
         self.n_head = n_head
-        self.w_qs = nn.Linear(dim_q, n_head * self.d_qk, bias=add_bias)  # q
-        self.w_ks = nn.Linear(dim_kv, n_head * self.d_qk, bias=add_bias)  # k
-        self.w_vs = nn.Linear(dim_kv, n_head * self.d_v, bias=add_bias)  # v
-        self.fc = nn.Linear(n_head * self.d_v, dim_q, bias=True)  # c
+        self.query = nn.Linear(dim_q, n_head * self.d_qk, bias=add_bias)  # q
```

Comment:
just confirming, the renaming is not breaking ckpt?

Reply:
It actually fixes the checkpoint